catalog/transformer/catalog-transformer-tika-input/src/test/java/ddf/catalog/transformer/input/tika/TikaInputTransformerTest.java

```diff
@@ -793,7 +793,9 @@ public void testMetadataExtractorMetacardType() throws Exception {
     addMetadataExtractor(metacardExtractor);
 
     Metacard metacard = transform(new ByteArrayInputStream("something".getBytes()));
-    assertThat(metacard.getMetacardType(), equalTo(metacardType));
+    assertThat(
```

Comment:
❓ Should we still keep the original assertion? If not, we should rename the test method.
❓ Can we be more specific in the new assertion, i.e., make sure the extracted text contains "something" instead of just using `notNullValue()`?

Reply:
I renamed the test and added an additional assertion.